draft-ietf-scitt-architecture.md

```diff
@@ -817,72 +818,234 @@ The last two steps may be shared between a batch of Signed Statements recorded i
 
-The Transparency Service may include the registration time to help Verifiers decide about the trustworthiness of the Transparent Statement.
 The registration time is defined as the timestamp at which the Transparency Service has added this Signed Statement to its Append-only Log.
 
+Editor's Note: The WG is discussing if existing CWT claims might better support these design principles.
+
 ~~~ cddl
-Receipt = [
-    version: int,
-    ts_identifier: tstr,
-    proof: SignedMerkleTreeProof
-]
 
-; Additional protected headers
-; in the COSE signed_tree_root of the SignedMerkleTreeProof
-Protected_Header = {
-  390 => int         ; SCITT Receipt Version
-  394 => tstr        ; DID of Transparency Service (required)
-  ? 393 => Reg_info  ; Registration policy information (optional)
+Inclusion_Proofs = [ + bstr ]
 
-  ; Other COSE Signed Merkle Tree headers
-  ; (e.g. tree algorithm, tree size)
+Verifiable_Data_Proof = {
+  &(inclusion-proof: -1) => Inclusion_Proofs
+}
 
-  ; Additional standard COSE headers
-  2 => [+ label]     ; Critical headers
-  ? 4 => bstr        ; Key ID (optional)
-  ? 33 => COSE_X509  ; X.509 chain (optional)
+Receipt_Unprotected_Header = {
+  &(proofs: -222) => Verifiable_Data_Proof
+}
+
+Receipt_CWT_Claims = {
+  1 => tstr; iss, the issuer signing the receipt (the identifier for the transparency service),
+  * tstr => any
 }
 
 ; Details of the registration info, as provided by the TS
 RegistrationInfo = {
   ? "registration_time": uint .within (~time),
   * tstr => any
 }
+
+Receipt_Protected_Header = {
+    -111 => int               ; Type of Verifiable Data Structure,
+                              ; for example "RFC9162_SHA256", or "CCF"
+
+    2 => [+ label]            ; Critical headers
+    13  => Receipt_CWT_Claims ; CBOR Web Token Claims
+    390 => int                ; SCITT Receipt Version
+
+    ? 4 => bstr        ; Key ID (optional)
+    ? 33 => COSE_X509  ; X.509 chain (optional)
+    ? 393 => Reg_info  ; Registration policy information (optional in Receipts)
+}
+
+Receipt_Headers = (
+    protected : *serialized* Receipt_Protected_Header,
+    unprotected : Receipt_Unprotected_Header
+)
+
+Receipt_as_COSE_Sign1 = [
+    Receipt_Headers,
+    payload : bstr / nil,
+    signature : bstr
+]
+
+Receipt = #6.18(Receipt_as_COSE_Sign1)
+
+; A Transparent Statement is a Signed Statement
+; with one or more Receipts in it's unprotected header.
+
+Signed_Statement_CWT_Claims = {
+  1 => tstr; iss, the issuer
+  1 => tstr; sub, the subject
+  * tstr => any
+}
+
+Signed_Statement_Headers = {
+    1 => int                            ; Signing Algorithm
+    13  => Signed_Statement_CWT_Claims  ; CBOR Web Token Claims
+    ? 4 => bstr         ; Key ID (optional)
+    ? 33 => COSE_X509   ; X.509 chain (optional)
+    393 => Reg_info     ; Registration policy information (mandatory in Signed_Statement)
```

Comment:
So this is a transitional version, where we have to decide what the difference between 13 and 393 is?

Reply:
Yep, not trying to fix everything at once.

Reply:
agreed, let's queue this up separately. @fournet was working on a new PR

Reply:
According to the current IANA registry https://www.iana.org/assignments/cose/cose.xhtml kccs is the header that has the correct type (map) and semantics (CWT claim set). So I changed the receipt type to use 14